xml/Microsoft.Extensions.FileSystemGlobbing/FilePatternMatch.xml

```diff
@@ -91,10 +91,10 @@
         <Parameter Name="other" Type="Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch" />
       </Parameters>
       <Docs>
-        <param name="other">The other match to be compared</param>
+        <param name="other">The other match to be compared.</param>
         <summary>Determines if the specified match is equivalent to the current match using a case-insensitive comparison.</summary>
         <returns>
-          <see langword="true" /> if <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Path" /> and <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Stem" /> are equal using case-insensitive comparison</returns>
+          <see langword="true" /> if <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Path" /> and <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Stem" /> are equal using case-insensitive comparison.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Path" /> and <see cref="P:Microsoft.Extensions.FileSystemGlobbing.FilePatternMatch.Stem" /> are equal using case-insensitive comparison; otherwise, <see langword="false" />.</returns>
```